katana/wapps/projects/templates/projects/display_suite.html

```diff
@@ -0,0 +1,67 @@
+<body>
```

Comment:
Is the file required ?

Reply:
Yes, File is required. Suite files should display in UI, after adding under suites section.